package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -0,0 +1,813 @@
+  - *donors* and *acceptors* atom types (to add additional atom names)
+
+.. _Analysis Output:
+
+Output
+------
+
+The results are
+  - the **identities** of donor and acceptor heavy-atoms from two selections
+  which are linked by a bridging water,
+  - the **distance** between the heavy atom acceptor atom and the hydrogen atom
+    that is bonded to the heavy atom donor,
+  - the **angle** donor-hydrogen-acceptor angle (180º is linear).
+
+Water bridge data are returned per frame, which is stored in
+:attr:`WaterBridgeAnalysis.timeseries` (In the following description, ``#``
+indicates comments that are not part of the output.)::
+
+    results = [
+        [ # frame 1
+           # hbonds linking the selection 1 and selection 2 to the bridgeing water 1
+           [ # hbond 1 from selection 1 to the bridging water 1
+              <donor index (0-based)>,
+              <acceptor index (0-based)>, <donor string>, <acceptor string>,
+              <distance>, <angle>
+           ],
+           [ # hbond 2 from selection 1 to the bridging water 1
+              <donor index (0-based)>,
+              <acceptor index (0-based)>, <donor string>, <acceptor string>,
+              <distance>, <angle>
+           ],
+           [ # hbond 1 from selection 2 to the bridging water 1
+              <donor index (0-based)>,
+              <acceptor index (0-based)>, <donor string>, <acceptor string>,
+              <distance>, <angle>
+           ],
+           [ # hbond 2 from selection 2 to the bridging water 1
+              <donor index (0-based)>,
+              <acceptor index (0-based)>, <donor string>, <acceptor string>,
+              <distance>, <angle>
+           ],
+
+           # hbonds linking the selection 1 and selection 2 to the bridgeing water 2
+           [ # hbond 1 from selection 1 to the bridging water 2
+              <donor index (0-based)>,
+              <acceptor index (0-based)>, <donor string>, <acceptor string>,
+              <distance>, <angle>
+           ],
+           [ # hbond 1 from selection 2 to the bridging water 2
+              <donor index (0-based)>,
+              <acceptor index (0-based)>, <donor string>, <acceptor string>,
+              <distance>, <angle>
+           ],
+           ....
+        ],
+        [ # frame 2
+          [ ... ], [ ... ], ...
+        ],
+        ...
+    ]
+
+Using the :meth:`WaterBridgeAnalysis.generate_table` method one can reformat
+the results as a flat "normalised" table that is easier to import into a
+database or dataframe for further processing.
+:meth:`WaterBridgeAnalysis.save_table` saves the table to a pickled file. The
+table itself is a :class:`numpy.recarray`.
+
+Detection of water bridges
+---------------------------
+Water bridges are recorded if a bridging water simultaneously form two hydrogen bonds
+with selection 1 and selection 2.
+
+Hydrogen bonds are recorded based on a geometric criterion:
```

Comment:
Link to hbond analysis!

Reply:
@orbeckst Thank you for the suggestion. Link created.